13-release/umbraco-cms/reference/content-delivery-api/extension-api-for-querying.md

```diff
@@ -230,18 +230,6 @@ The principal difference from the selector is that the filter implements `BuildF
 
 Since we need to perform an exact match, the index field (`authorId`) is once again defined as a "raw" string. Other options include "analyzed" and "sortable" strings. These support "contains" searches and alpha-numeric sorting, respectively.
 
-### Filter operations
-
-To define your custom filter, you currently have four options: `Is`, `IsNot`, `Contains` and `DoesNotContain`.
-
-It is possible to pass multiple values to each operator, and these values will be treated inclusively as an __or__ operator. For example, if `tag1` and `tag2` were passed into a filter using the `Is` operator, _any_ document containing __either__ `tag1` __or__ `tag2` would return. The request for this might look like this:
-
-`GET /umbraco/delivery/api/v1/content?filter=customTagFilter:tag1,tag2`
-
-If you require this functionality to be restrictive i.e. `tag1` __and__ `tag2`, then the current approach would be to chain the custom filter. The request would change to look more like this:
-
-`GET /umbraco/delivery/api/v1/content?filter=customTagFilter:tag1&filter=customTagFilter:tag2`
-
```

Comment:
is this supposed to be removed? I can see on this PR: https://github.com/umbraco/UmbracoDocs/pull/5682/files#diff-f64d504745f1560b024cf3cf71341bb4d8668075d0c8d0d6964c8a7ccc77acab that I am gonna merge thurdsday that things are added on top of this 🤔

Reply:
Good catch!

I have added it back :D